packages/calcite-components/src/components/label/label.e2e.ts

```diff
@@ -145,6 +146,44 @@ describe("calcite-label", () => {
     expect(element).toEqualAttribute("layout", "inline-space-between");
   });
 
+  it("should focus on the slotted form element when a label is clicked", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-label id="do-not-duplicate-me" layout="inline-space-between">
```

Comment:
Sorry for not spotting this earlier, but let's remove anything unrelated to the test. This `id` comes from a test specific to that prop/attr.